.github/workflows/main.yml

```diff
@@ -91,11 +91,16 @@ jobs:
         restore-keys: ${{ runner.os }}-gradle
     - name: Build with Maven
       run: ./mvnw -B deploy --file pom.xml -Pcode-coverage,jdk8-tests,native,release -DdeployAtEnd=true -Dtest.log.level=WARN
+      if: ${{ github.repository == 'projectnessie/nessie' }}
```

Comment:
how about checking if MAVEN_USERNAME is defined?

Reply:
Yup, changed to that in the whole PR.